sdk/dotnet/test/Microsoft.AspNetCore.Datasync.Test/Extensions/IHeaderDictionary.Test.cs

```diff
@@ -105,7 +106,7 @@ public void AddFromEntity_BothHeaders_WhenBothSet()
             Assert.Single(headers["ETag"]);
             Assert.Equal(testETag, headers["ETag"][0]);
             Assert.Single(headers["Last-Modified"]);
-            Assert.Equal("Wed, 30 Jan 2019 13:30:15 GMT", headers["Last-Modified"][0]);
+            Assert.Equal(DateTimeOffset.Parse("2019-01-30T13:30:15Z").ToString(DateTimeFormatInfo.InvariantInfo.RFC1123Pattern), headers["Last-Modified"][0]);
```

Comment:
I'm not sure the DateTimeOffset.Parse ones are required - I'm doing a specific string that is already in that format.

Reply:
because if I leave the english string, test fail:
```
Expected: Wed, 30 Jan 2019 13:30:15 GMT
Actual:   mer, 30 gen 2019 13:30:15 GMT
```
instead parsing the datetime to invariant rfc pattern use the same language:
it CultureInfo
```
Expected: mer, 30 gen 2019 13:30:15 GMT
Actual:   mer, 30 gen 2019 13:30:15 GMT
```
en CultureInfo
```
Expected: Wed, 30 Jan 2019 13:30:15 GMT
Actual:   Wed, 30 Jan 2019 13:30:15 GMT
```

Reply:
Is it coming across the wire as italian?  (Which would be a problem in the server where we generate the header).  I'm comparing what is coming across the wire with the expected value.  Last-Modified header should be invariant culture.  This is the line that produces that header: https://github.com/Azure/azure-mobile-apps/blob/main/sdk/dotnet/src/Microsoft.AspNetCore.Datasync/Extensions/IHeaderDictionaryExtensions.cs#L30.  

This feels like a test fail in a culture sensitive way.

I need to set up an environment this week which can take a look at the effect of culture within the service.

Reply:
I think what we need to do is alter IHeaderDictionaryExtensions.cs to be ...ToString(DateTimeInfoFormat.InvariantCulture.RFC1123Format, CultureInfo.InvariantCulture) - and then the test will pass.

Reply:
> I think what we need to do is alter IHeaderDictionaryExtensions.cs to be ...ToString(DateTimeInfoFormat.InvariantCulture.RFC1123Format, CultureInfo.InvariantCulture) - and then the test will pass.

add CultureInfo.InvariantCulture in string conversion ... I'm not a globalization expert :-)

Reply:
I have to admit I had to consult a globalization expert myself :-)  This is a learning experience kind of day.